CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Support for stdoutlog exporter in `go.opentelemetry.io/contrib/config`. (#5850)
 - Add macOS ARM64 platform to the compatibility testing suite. (#5868)
+- The `go.opentelemetry.io/contrib/bridges/otelzap` module.
+  This module provides an OpenTelemetry logging bridge for "go.uber.org/zap". (#5191)
```

Comment:
```suggestion
  This module provides an OpenTelemetry logging bridge for `go.uber.org/zap`. (#5191)
```

Reply:
Usually, we link to the PR doing the implementation rather than a related issue. But in your case, there are 21 PRs implementing this bridge. So linking to a single issue makes sense.